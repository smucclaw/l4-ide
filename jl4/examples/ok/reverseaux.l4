go acc l MEANS
  CONSIDER l
  WHEN EMPTY            THEN acc
  WHEN x FOLLOWED BY xs THEN go (x FOLLOWED BY acc) xs

#CHECK go
