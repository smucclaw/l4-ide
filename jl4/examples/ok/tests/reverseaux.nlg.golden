`go`
Info | updateFileDiagnostics published different from new diagnostics - file diagnostics:
File: reverseaux.l4
  Hidden:   no
  Range:    6:8-6:10
  Source:   check
  Severity: DiagnosticSeverity_Information
  Code:     <none>
  Message:  FUNCTION FROM LIST OF p8 AND LIST OF p8 TO LIST OF p8 